cypress/support/pageObjects/performancePageLocators.js

```diff
@@ -0,0 +1,335 @@
+class performancePageLocators {
+    //Locators
+    _linkPrefetchText = '.newfold-link-prefetch';
```

Comment:
```suggestion
     getlinkPrefetchText() {
     return cy.get('.newfold-link-prefetch');
  }
```
Would it be better to use getter methods instead of storing locators as variables? That way, it’s easier to maintain and it makes sure that evaluation is always the latest when Cypress re-renders.